test/e2e/workflow/deployment_test.go

```diff
@@ -128,6 +144,11 @@ var _ = Describe("NetworkAddonsConfig", func() {
 			configSpec.MacvtapCni = &cnao.MacvtapCni{}
 			components = append(components, MacvtapComponent)
 			testConfigUpdate(gvk, configSpec, components)
+
+			// Add Multus Dynamic Networks component (requires multus ...)
+			configSpec.Multus = &cnao.Multus{}
+			configSpec.MultusDynamicNetworks = &cnao.MultusDynamicNetworks{}
+			testConfigUpdate(gvk, configSpec, append(components, MultusComponent, MultusDynamicNetworks))
```

Comment:
Following the existing structure where `components` are carried over to the next component will make it easier to add a new component in the future

```suggestion
			configSpec.MultusDynamicNetworks = &cnao.MultusDynamicNetworks{}
			components = append(components, MultusDynamicNetworks, MultusComponent)
			testConfigUpdate(gvk, configSpec, components))
```

Reply:
OK, I'll keep using this mutato.